docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -43,61 +43,121 @@ Agent
 Agents represent owners or custodian of goods in the system. All agents must
 be registered in the system before they can interact with records.
 
-.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
```

Comment:
Probably true if anyone checks. In this case there were offset errors which have gone unnoticed for at least a few weeks. It's also only obvious if you build the docs. You cannot tell from the source code.

I'm also not sure including the exact source code is important. People can always look that up if they need it. A doc should have enough example code to explain how to use a component. Minor inaccuracies may not impair that use case. And regardless of whether it is in the example code, or elsewhere in the doc, you will always have the headache of making sure engineers update documentation when they change functionality.